Detectors/TPC/calibration/SpacePoints/include/SpacePoints/TrackInterpolation.h

```diff
@@ -0,0 +1,243 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file TrackInterpolation.h
+/// \brief Definition of the TrackInterpolation class
+///
+/// \author Ole Schmidt, ole.schmidt@cern.ch
+
+#ifndef ALICEO2_TPC_TRACKINTERPOLATION_H_
+#define ALICEO2_TPC_TRACKINTERPOLATION_H_
+
+#include "CommonDataFormat/EvIndex.h"
+#include "ReconstructionDataFormats/Track.h"
+#include "ReconstructionDataFormats/TrackTPCITS.h"
+#include "ReconstructionDataFormats/MatchInfoTOF.h"
+#include "DataFormatsITSMFT/Cluster.h"
+#include "DataFormatsITS/TrackITS.h"
+#include "DataFormatsTPC/ClusterNativeHelper.h"
+#include "DataFormatsTPC/TrackTPC.h"
+#include "DataFormatsTPC/Constants.h"
+#include "DataFormatsTOF/Cluster.h"
+#include "SpacePoints/SpacePointsCalibParam.h"
+#include "TPCReconstruction/TPCFastTransformHelperO2.h"
+
+class TTree;
+
+namespace o2
+{
+namespace tpc
+{
+
+/// Intermediate structure filled with TPC position information based in ITS-TOF interpolation (per track)
+struct CacheMeanPosInTPC {
+  std::array<std::array<float, 2>, Constants::MAXGLOBALPADROW> posExt{}; ///< y and z coordinate for extrapolated ITS tracks
```

Comment:
Yes, that is because I was wrongly identifying extrapolation with something outwards and interpolation with something inwards. I changed it to something clearer now.